syntax = "proto3";

package xlerion;

message Empty {}

message GraphDataPoint {
  string label = 1;
  float value = 2;
}

message GraphDataResponse {
  repeated GraphDataPoint data_points = 1;
}

service DataProcessingService {
  rpc GetGraphData (Empty) returns (GraphDataResponse) {}
}
